syntax = "proto2";

package grr;

import "grr_response_proto/semantic.proto";

message ApiKillFleetspeakArgs {
  optional string client_id = 1 [(sem_type) = { type: "ClientURN" }];
  // If true, the fleetspeak process is terminated without performing
  // a graceful shutdown.
  optional bool force = 2;
}

message ApiRestartFleetspeakGrrServiceArgs {
  optional string client_id = 1 [(sem_type) = { type: "ClientURN" }];
}
